.devcontainer/resources/k3s.service

```diff
@@ -0,0 +1,27 @@
+[Unit]
```

Comment:
Remove k3s-related resources since it has been removed from the Leda distro?

Reply:
Removed